### DISCUSSION TITLE
Update link in repository "about" section

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#15736] [Suggest an Idea]
Steps to reproduce:

1. Go to [`renovatebot/renovate` homepage](https://github.com/renovatebot/renovate)
2. Go the sidebar on the right, the "About" section
3. Click on the link
4. End up on the new `mend.io` domain, after a redirect from the browser

**Comment by rarkins** [comment]
I'm fine with this being a redirect. The URL on mend may change sometime in future, but the redirect won't
